ntbs-service/Pages/Admin/Migration.cshtml

```diff
@@ -1,105 +1,104 @@
-                <label nhs-label-type="Standard" for="notification-end-date"> End Notification Date Range </label>
-                <nhs-date-input id="notification-date">
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Day">Day</label>
-                            <input nhs-input-type="Date" fixed-width="Two"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Day" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Month">Month</label>
-                            <input nhs-input-type="Date" fixed-width="Two"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Month" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Year">Year</label>
-                            <input nhs-input-type="Date" fixed-width="Four"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Year" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                </nhs-date-input>
-            </nhs-form-group>
-        </div>
-
-        <div>
-            <button nhs-button-type="Standard" type="submit" value="Import" classes="ntbsuk-button--primary submit-search-button" id="search-button">
-                Import
-            </button>
-        </div>
-    </form>
-
-    <br/>
-    <a href='/Hangfire/jobs/succeeded'> Navigate to dashboard </a>
+@page
+@model ntbs_service.Pages.Admin.MigrationModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    ViewData["Title"] = "Migration";
+    Layout = "Shared/_Layout";
+}
+
+<nhs-width-container container-width="Standard">
+    <h1>Import Legacy Notifications</h1>
+
+    <form method="post" enctype="multipart/form-data">
+        <input nhs-input-type="Standard" asp-for="NotificationIds">
+        <br/>
+
+        <div>
+            <nhs-form-group>
+                <label asp-for="UploadedFile">Upload File</label>
+                <input type="file" asp-for="UploadedFile"/>
+            </nhs-form-group>
+        </div>
+        <br/>
+
+        <div>
+            @{
+                var notificationDateStartError = !Model.ValidationService.IsValid("NotificationDateRangeStart");
```

Comment:
I'm a big fan of using `nameof()` for these now - settled on it in the last few page changes I did.

Reply:
as in
`var notificationDateEndError = !Model.ValidationService.IsValid(nameof(Model.NotificationDateRangeEnd));`? looks pretty sweet :)